lib/node/rpc.js

```diff
@@ -485,11 +485,13 @@ class RPC extends RPCBase {
 
       peers.push({
         id: peer.id,
-        addr: peer.fullname(),
+        addr: peer.realname(),
         addrlocal: !peer.local.isNull()
           ? peer.local.fullname()
           : undefined,
         name: peer.name || undefined,
+        encrypted: peer.encrypt,
+        identitykey: peer.address.getKey('base32'),
```

Comment:
For clearnet peers, this will display the always-hilarious bech32 string:
` identitykey: 'aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa',`

Reply:
Do you think it should return `null` instead?

Reply:
Maybe, or not display anything. I guess `null` is better. Could this be written in the actual `NetAddress.getKey()` function to return null if it equals `ZERO_KEY` ?

Reply:
Updated this functionality to return `null` if there is no key